docs/working_with_page.md

```diff
@@ -0,0 +1,226 @@
+!!! enterprise
+
+    # Working with a Page
+    
+    **Page** is a special Content Type that contains zones onto which you can drop different dynamic blocks.
+    You create and edit Pages using the Page Builder accessible in the Back Office by clicking **Page**
+    and selecting the page to edit from the toolbar that appears below.
+    You can create a Content item here by clicking the plus button in the upper-right corner.
+    
+    !!! tip
+        The Page Content Type contains a **Landing Page** Field Type which manages the zones and blocks.
+        Any Content item that has the **Landing Page** Field Type will behave like the Page Content Type.
+    
+    ## Getting started
+    
+    The Page Builder comes with a toolbar that enables you to perform advanced editing and previewing of the Page.
+    
+    ![Page Builder Toolbar](img/page_builder_toolbar.png)
+    
+    Using the toolbar buttons, you can perform the following actions:
+    
+    |Button|Description|
+    |------|-----------|
+    |![](img/page_builder_toolbar_editpreview.png)|Switch between previewing and editing the Page.|
+    |![](img/page_builder_toolbar_viewedit.png)|Switch between viewing the Page contents and the Page title and description.|
+    |![](img/page_builder_toolbar_devicestoggler.png)|Toggle through to see how the Page is rendered on different devices.|
+    |![](img/page_builder_toolbartimelinetoggler.png)|Access the timeline to preview the Page changes depending on the date or view the list of all upcoming scheduled events. For details, see [Timeline](publishing/advanced_publishing_options.md#timeline).|
```

Comment:
`Access the timeline...` This sentence is too convoluted. How about something like `Access the timeline to preview how the Page changes with time. You can also view...`